examples/python/twoday_p.py

```diff
@@ -83,68 +88,85 @@ def run(plot=False, customized_kpi_config=None):
                customizedkpis_result[kpicalculation.cutomizedKPI(config[key]).name]=[]
     customizedkpis_result['time']=[]           
     # --------------------
-    
+
+
     # RUN TEST CASE
     # -------------
-    # Reset test case
-    print('Initializing the simualation.')
-    res = requests.put('{0}/initialize'.format(url), data={'start_time':0,'warmup_period':0})
-    if res:
+    start = time.time()
+    # Initialize test case
+    print('Initializing test case simulation.')
+    res = requests.put('{0}/initialize'.format(url), json={'start_time':0,'warmup_period':0}).json()
+    if res['message'] == 'success':
         print('Successfully initialized the simulation')
-    # Set simulation step
-    print('Setting simulation step to {0}.'.format(step))
-    res = requests.put('{0}/step'.format(url), data={'step':step})
+    else:
+        print('Error when initializing the simulation:{}'.format(res['error']))    
     print('\nRunning test case...')
+    # Set simulation step
+    res = requests.put('{0}/step'.format(url), json={'step':step}).json()
+    if res['message'] == 'success':
+        print('Successfully set the simulation step')
+    else:
+        print('Error when setting the simulation step:{}'.format(res['error']))   
+
     # Initialize u
     u = pid.initialize()
     # Simulation Loop
     for i in range(int(length/step)):
         # Advance simulation
-        y = requests.post('{0}/advance'.format(url), data=u).json()
+        y = requests.post('{0}/advance'.format(url), json=u).json()
```

Comment:
I suggest using "res" instead of "y" and then later assigning "y" as the res["result"], to keep consistent.

Reply:
Sure, will do